generators/server/templates/src/main/docker/_Dockerfile

```diff
@@ -20,11 +20,14 @@ FROM <%= DOCKER_JAVA_JRE %>
 
 ENV SPRING_OUTPUT_ANSI_ENABLED=ALWAYS \
     JHIPSTER_SLEEP=0
+ENV JAVA_OPTS=""
```

Comment:
it should be in the same line ENV above, so it won't add additional Docker layer